filebeat/reader/multiline/multiline.go

```diff
@@ -66,7 +66,7 @@ const (
 type matcher func(last, current []byte) bool
 
 var (
-	sigMultilineTimeout = errors.New("multline timeout")
+	sigMultilineTimeout = errors.New("multiline timeout")
```

Comment:
Technically this would probably change how consumers experience the exception, but only if they look at the error string...